### PR TITLE
Fix paths in tests

### PR DESCRIPTION
**Pull request by shlyakpavel**
Each commit addresses a test or a set of test that caused "file not found" errors unintentionally

### CODE REVIEW

**Comment by shlyakpavel** [comment]
Also, 

- [ ] Text/input/wpt-import/css/selectors/focus-visible-009.html cannot find utils.js
- [ ] Text/input/wpt-import/css/selectors/focus-visible-017-2.html cannot find green.png

**Comment by AtkinsSJ** [comment]
Did you fix these manually or delete them and run the importer again? I made the importer a bit smarter recently but if there are still things it misses it'd be good to know.

**Comment by shlyakpavel** [comment]
@AtkinsSJ I tried deleting and reimporting, but that didn't help me, so I had to do it manually. Also, some of the tests I fixed here are actually not WPT tests.

**Comment by shlyakpavel** [comment]
Rebased to fix CI

**Comment by shlyakpavel** [comment]
CI failure is still totally unrelated to the issue, timing out just like it does on other branches and PRs

File: Tests/LibWeb/Layout/input/input-text-to-image.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<input type="text" src="120.png" width="120" height="120" />
+<input type="text" src="../../../LibWeb/Assets/120.png" width="120" height="120" />
```
Comment (AtkinsSJ) [inline Tests/LibWeb/Layout/input/input-text-to-image.html:1]: You can drop the `../LibWeb/` from this, we're already in `LibWeb`

File: Tests/LibWeb/Text/input/wpt-import/css/css-grid/alignment/grid-alignment-implies-size-change-011.html [hunk 2]
```diff
@@ -49,6 +49,6 @@
 </script>
 <body onload="document.fonts.ready.then(() => { runTest(); })">
 <div class="grid" id="grid">
-  <img id="item" src="support/100x100-green.png"></img>
+  <img id="item" src="../../css-flexbox/support/100x100-green.png"></img>
```
Comment (AtkinsSJ) [inline Tests/LibWeb/Text/input/wpt-import/css/css-grid/alignment/grid-alignment-implies-size-change-011.html:52]: This isn't the correct URL. The tests on wpt.live are linking to `https://wpt.live/css/css-grid/alignment/support/100x100-green.png`. The real issue seems to be that we don't have that file downloaded.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
The first two commits don't affect the test results at all, what's happening with those?

**Comment by shlyakpavel** [comment]
Maybe I should just wait for https://github.com/LadybirdBrowser/ladybird/issues/2282

**Comment by shlyakpavel** [comment]
> The first two commits don't affect the test results at all, what's happening with those?

That's ok, we just don't pass them. Now there's no 404 in the log, so this is a small step towards these tests passing in the future when related implementation improves in Ladybird

**Comment by shlyakpavel** [comment]
Since I made these PR, these tests introduced new File not founds:
- A lot of tests under Tests/LibWeb/Text/input/wpt-import/html/webappapis/dynamic-markup-insertion/document-write
- Text/input/wpt-import/html/dom/aria-attribute-reflection.html
- Text/input/wpt-import/dom/events/Event-constants.html (it's main js file lol)
- Text/input/wpt-import/custom-elements/upgrading.html

This spreads like a virus...

**Comment by shlyakpavel** [comment]
Whoever reviews this, please just run 
`./Meta/ladybird.sh run headless-browser -R Tests/LibWeb -j1`
and compare the before/after for yourself

**Review by AtkinsSJ — APPROVED** [review]
Thanks for this!

**Pull request merged**